php/meterpreter/ext_server_stdapi.php

```diff
@@ -452,7 +452,8 @@ function resolve_host($hostname, $family) {
     } elseif ($family == AF_INET6) {
         $dns_family = DNS_AAAA;
     } else {
-        throw new Exception('invalid family, must be AF_INET or AF_INET6');
+        my_print('invalid family, must be AF_INET or AF_INET6');
+        return ERROR_FAILURE;
```

Comment:
Looks like this needs a tweak still, since it's meant to return null or an object

Reply:
I'll make this return NULL on landing